tests/functional/setups/manu_thermoporomech_nofrac_2d.py

```diff
@@ -127,7 +127,7 @@ class ManuThermoPoroMechDataSaving(VerificationDataSaving):
     :class:`~porepy.models.fluid_mass_balance.VariablesSinglePhaseFlow`.
 
     """
-    displacement: Callable[[list[pp.Grid]], pp.ad.MixedDimensionalVariable]
+    displacement: Callable[[pp.SubdomainsOrBoundaries], pp.ad.MixedDimensionalVariable]
```

Comment:
Have you checked that there are no more declarations in the derived model classes in this and related files which can be purged?

Reply:
No. Me and Eirik agreed that they can be found and replaced gradually, when revisited. @keileg please correct me if I'm wrong.

Reply:
Yes, I agree that improving the type declarations in these setups (which are never checked by mypy, btw) is not a prioritized task. It can be argued that the declarations should be deleted from these setups, but that is a different question.